github/localrepo.go

```diff
@@ -191,14 +191,14 @@ func (r *GitHubRepo) MainRemote() (remote *Remote, err error) {
 func (r *GitHubRepo) MainProject() (project *Project, err error) {
 	origin, err := r.MainRemote()
 	if err != nil {
-		err = fmt.Errorf("Aborted: the origin remote doesn't point to a GitHub repository.")
+		err = fmt.Errorf("Aborted: the origin remote doesn't point to a GitHub repository:\n %v.", err)
```

Comment:
There's no need to indent the following line, since those two lines will likely be the only lines of stderr output from the program, which will abort at this point. There will be no confusion that the 2nd line refers to the 1st. Also, the first line should end in a period so that it forms a sentence and doesn't have the colon twice.